lnwallet/channel.go

```diff
@@ -6354,20 +6355,20 @@ func (lc *LightningChannel) ReceiveHTLCSettle(preimage [32]byte, htlcIndex uint6
 // testing the wallet.
 func (lc *LightningChannel) FailHTLC(htlcIndex uint64, reason []byte,
 	sourceRef *channeldb.AddRef, destRef *channeldb.SettleFailRef,
-	closeKey *models.CircuitKey) error {
+	closeKey *models.CircuitKey) (*PaymentDescriptor, error) {
```

Comment:
In the last commit, we added a way to look up the blinding point for a given HTLC index, can that be used again instead of extending this API?